### DISCUSSION TITLE
Numpy not being detected

### DISCUSSION THREAD

**Discussion by dghjunior** [khoj-ai/khoj#178] [Q&A]
![Screenshot 2023-02-21 at 3 25 43 AM](https://user-images.githubusercontent.com/56708171/220289061-8499e27b-c03b-40bd-a095-efc82024169e.jpg)
As you can see in the image here, I get an error when trying to run khoj to be used as Obsidian plugin, but it doesn't detect my numpy install. I even tried installing numpy 1.17 and it didn't work either. Suggestions to fix?

**Comment by debanjum** [comment]
Strange 🤔. Maybe an issue with different python, pip or package install directory being used to install `numpy` vs `khoj`?
- Could you share logs/screenshots of you installing `khoj` with `pip install khoj-assistant` and numpy with `pip install numpy`? It would be good to see the `numpy` install directory mentioned by the khoj and numpy install logs
- If you're in a hurry to try khoj (i.e before we've resolve this issue), installing khoj in a virtualenv maybe the fastest route. For e.g with [pipx](https://pypa.github.io/pipx/#install-pipx) by running `pip install pipx && pipx install khoj`

**Comment by debanjum** [comment]
Converting this into a discussion until any concrete issues in khoj are identified due to this investigation
